### PR TITLE
parse 1.3.2

### PR DESCRIPTION
**Pull request by n0ts**
Add parse.com command line tools

### CODE REVIEW

**Comment by mistydemeo** [comment]
Sorry for the late reply.

Do you mind requesting a versioned download URL from the developer?

**Comment by n0ts** [comment]
Parse download URL is public.
https://parse.com/docs/cloud_code_guide

URL "https://www.parse.com/downloads/cloud_code/parse" is included public installer.sh.

Thanks.

**Comment by mistydemeo** [comment]
Sorry, I mean a URL that downloads a specific version. That would let the formula always point to a specific version of parse, which is how most formulae work.

**Comment by n0ts** [comment]
I change version is "latest".

File: parse.rb [hunk 1]
```diff
@@ -0,0 +1,17 @@
+require "formula"
+
+class Parse < Formula
+  homepage "https://parse.com"
+  url "https://www.parse.com/downloads/cloud_code/parse", :using => :nounzip
+  version :latest
+  sha256 :no_check
```
Comment (jacknagel) [inline parse.rb]: Homebrew doesn't support this.

**Comment by n0ts** [comment]
I pull is closed.
Thanks.
parse download URL is not included specific version.

**Pull request closed without merging**